lib/yard-sorbet/struct_handler.rb

```diff
@@ -41,10 +41,14 @@ def process
 
     # Register the object explicitly as an attribute
     namespace.attributes[scope][name][:read] = object
+
+    # While `const` attributes are immutable, `prop` attributes may be reassigned.
+    method_name = statement.method_name.source
+    namespace.attributes[scope][name][:write] = object if method_name == 'prop'
```

Comment:
It's a bit easier to read imho as a normal `if`:
```suggestion
    if statement.method_name.source == 'prop'
      namespace.attributes[scope][name][:write] = object 
    end
```